monitor.py

```diff
@@ -74,6 +72,9 @@
 if not os.path.exists(args.backup_path):
     os.mkdir(args.backup_path)
 
+# List of public workers to monitor
+public_workers = os.environ['CODALAB_PUBLIC_WORKERS'].split(',')
```

Comment:
Be more explicit that these are worker ids?  But the worker ids are assumed to be hosts here and thus stable?  At some point they had hosts + a unique ID.  Make this explicit.

Reply:
Updated! The env variable `CODALAB_PUBLIC_WORKERS` will be a string containing workers_id `'vm-clws-prod-worker-0` separated by ','.